File: .tmp_py4013/batch_11_dump/3295_report_spam_message.js
// LeetCode 3295 - Report Spam Message
// https://leetcode.com/problems/report-spam-message/

var reportSpam = function(message, bannedWords) {
    const ban = new Set(bannedWords);
    let cnt = 0;
    for (const w of message) {
        if (ban.has(w)) {
            cnt++;
            if (cnt >= 2) return true;
        }
    }
    return false;
};

File: .tmp_rs4013/rust_check16/3295_report_spam_message.rs
struct Solution;
// LeetCode 3295 - Report Spam Message
// https://leetcode.com/problems/report-spam-message/

use std::collections::HashSet;

impl Solution {
    pub fn report_spam(message: Vec<String>, banned_words: Vec<String>) -> bool {
        let ban: HashSet<String> = banned_words.into_iter().collect();
        let mut cnt = 0;
        for w in message {
            if ban.contains(&w) {
                cnt += 1;
                if cnt >= 2 {
                    return true;
                }
            }
        }
        false
    }
}

fn main() {}
